common/app/model/DatesAndTimes.scala

```diff
@@ -97,9 +97,9 @@ object GUDateTimeFormatNew {
   }
   def formatDateTimeForDisplayGivenEdition(date: DateTime, edition: Edition): String = {
     val timezone = edition.timezone
-    date.toString(DateTimeFormat.forPattern("E d MMM yyyy HH.mm").withZone(timezone)) + " " + timezone.getShortName(
-      date.getMillis,
-    )
+    val timeZoneString1 = timezone.getShortName(date.getMillis)
+    val timeZoneString2 = if (timeZoneString1 == "GMT-04:00") "EDT" else timeZoneString1
+    date.toString(DateTimeFormat.forPattern("E d MMM yyyy HH.mm").withZone(timezone)) + " " + timeZoneString2
```

Comment:
What is the motivation for these changes? I wasn't sure how this differs from the previous code and why. Improving naming here to something more descriptive for `timeZoneString1/2` might help to.

Reply:
Oh, yeah. I was about to describe that change in the PR description. 

For some reasons `timezone.getShortName(date.getMillis)` no longer returns "EDT" as it used to do, but now returns "GMT-04:00". So I added a line of code. 

I get your point about the naming, let me update that :)